_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,136 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements within a given [document context](#document-context) that have either `title`, `aria-label` or `aria-labelled-by` attribute.
+
+**Note: This rule does not ensure that the `iframe` targets with the same `src` destination have the same title. It only checks `title` of a given frame for uniqueness across the document context.**
+
+### Expectation
+
+Each test target has a `title`, `aria-label` or `aria-labelledby` attribute value that is unique within the [document context](#document-context) of the element.
+
+ ## Assumptions
+
+The rule assumes that the target `iframe` is used as an user interface component, unless explicity overridden by specifying `tabindex="-1"`.
```

Comment:
Why does this matter? It seems to me if we want to ignore iframes with tabindex (which I don't know we should) we can do it in the rule, not put it on the assumptions.